CODEOWNERS

```diff
@@ -1,6 +1,13 @@
 # These datasets are subject to the additional change control procedures
 # described in https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/
 # Active Users
+/sql_generators/active_users_aggregates_v3/templates/ @mozilla/kpi_table_reviewers
+/sql/moz-fx-data-shared-prod/fenix_derived/active_users_aggregates_v3/ @mozilla/kpi_table_reviewers
+/sql/moz-fx-data-shared-prod/firefox_desktop_derived/active_users_aggregates_v3/ @mozilla/kpi_table_reviewers
```

Comment:
Could we not just use pattern matching here instead of having abound here for each product considering the same owner is assigned? Here's an example:

https://www.curiouslychase.com/posts/github-codeowners-directory-pattern-match-cheatsheet